src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/SelfSignAccessRequirementWidget.java

```diff
@@ -103,6 +106,10 @@ public void onSuccess(String rootWikiId) {
 		subjectsWidget.configure(ar.getSubjectIds());
 		manageAccessButton.configure(ar);
 		lazyLoadHelper.setIsConfigured();
+		view.setAccessRequirementID(ar.getId().toString());
+		isACTMemberAsyncHandler.isACTActionAvailable(isACT -> {
+			view.setAccessRequirementIDVisible(isACT);
+		});
```

Comment:
Seems like setIsConfigured should be called last.
```suggestion
		view.setAccessRequirementID(ar.getId().toString());
		isACTMemberAsyncHandler.isACTActionAvailable(isACT -> {
			view.setAccessRequirementIDVisible(isACT);
		});
		lazyLoadHelper.setIsConfigured();
```